### PR TITLE
New version: Microsoft.PIX 2409.23

### PR DESCRIPTION
**Pull request by autumn6liu**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue? https://github.com/microsoft/winget-pkgs/issues/175495

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/175496)

### CODE REVIEW

**Comment by autumn6liu** [comment]
@microsoft-github-policy-service agree company="Microsoft"

**Comment by autumn6liu** [comment]
There is no "AppsAndFeaturesEntries" property when running the current version of YamlCreate

**Comment by stephengillie** [comment]
Hi @autumn6liu,

This manifest removes the DisplayVersion from the AppsAndFeaturesEntries, which is present in previous manifest versions. This entry should be added to this version, to maintain version matching, and prevent the 'upgrade always available' situation with this package.

(Automated response - build 896.)

File: manifests/m/Microsoft/PIX/2409.23/Microsoft.PIX.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Created with YamlCreate.ps1 v2.4.1 $debug=NVS1.CRLF.5-1-22621-4111.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Microsoft.PIX
+PackageVersion: "2409.23"
+InstallerType: burn
+Scope: machine
```
Comment (a-mnich) [inline manifests/m/Microsoft/PIX/2409.23/Microsoft.PIX.installer.yaml:7]:
```suggestion
Scope: machine
AppsAndFeaturesEntries:
  - DisplayName: Microsoft PIX 2409.23
    DisplayVersion: 24.9.23.1
```

**Review by a-mnich — CHANGES_REQUESTED** [review]
I've requested a change that add the missing AppsAndFeatures entry.  

Verification:
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**